### ISSUE TITLE
OpenTK window not opening when running sample on linux

### ISSUE THREAD

**Issue by SpaceCats64** [mellinoe/ImGui.NET#20]
I'm not sure if this is just an issue with using archlinux or not as you said you were able to get it running on ubuntu, but when running the provided sample program using the dotnet tools it doesn't crash but it doesn't display an OpenTK window either.

**Comment by mellinoe** [comment]
Hm, interesting. If the window itself isn't even showing up then I would have to guess that there's a problem with OpenTK. Do you have SDL2 installed? I've found that often OpenTK behaves a little better in some cases if it's using the SDL2 backend. Not sure if SDL2 is available on Arch Linux or not.

Do you know if the program is reaching any of the real drawing code, or is it hanging before that or something?

**Comment by SpaceCats64** [comment]
I was actually able to get the window to pop up for a second by adding `using System` to Program.cs (I added it so I could print some stuff to the console to test where things were hanging). 

Sill it only pops up for a second throwing a System.DllNotFound exception. Heres the error message i'm getting.
```
Unhandled Exception: System.TypeInitializationException: The type initializer for 'ImGuiNET.ImGui' 
threw an exception. ---> System.DllNotFoundException: Unable to load DLL 'cimgui': The specified module could not be found.
 (Exception from HRESULT: 0x8007007E)
   at ImGuiNET.ImGuiNative.igGetIO()
   at ImGuiNET.ImGui..cctor() in /home/spacey/ImGui.NET/src/ImGui.NET/ImGui.cs:line 25
   --- End of inner exception stack trace ---
   at ImGuiNET.ImGui.GetIO() in /home/spacey/ImGui.NET/src/ImGui.NET/ImGui.cs:line 27
   at ImGuiNET.SampleWindow..ctor() in /home/spacey/ImGui.NET/src/ImGui.NET.SampleProgram/SampleWindow.cs:line 53
   at ImGuiNET.Program.Main(String[] args) in /home/spacey/ImGui.NET/src/ImGui.NET.SampleProgram/Program.cs:line 8
```
I think I may have forgot to build cimgui for myself, anyway thanks for the help!

**Comment by mellinoe** [comment]
Yep, you'll need to do that, to start. You should use commit fb044c4b6a25a5f8ce6409d6ecfc8702828b6e65 from cimgui.

https://github.com/Extrawurst/cimgui/commit/fb044c4b6a25a5f8ce6409d6ecfc8702828b6e65

**Comment by SpaceCats64** [comment]
After building it myself (trying both that specific commit and the latest version) it seems to only _sometimes_. For sometimes it hangs on creating the window, sometimes it segfaults, sometimes it crashes with no message, and sometimes it works. Its hard to exaplain; but it _does_ seem to be an issue with cimgui and not ImGui.NET.

**Comment by mellinoe** [comment]
Hm, the fact that window creation doesn't work sometimes makes me believe that it's a problem with OpenTK. It's interesting that it sometimes works, though...

**Comment by SpaceCats64** [comment]
Updated all of my packages and it works 99% of the time now. Still not sure why it opens then insta closes that 1% of the time but it isnt much of an issue. Thanks for your help man!

**Comment by mellinoe** [comment]
Nice! If you get any more information about that 1%, I can try to help. Glad you've got things working most of the time, though.

**Comment by dpethes** [comment]
Maybe you could try with newer version of OpenTK? Iirc it was quite easy to switch to 2.0.

**Comment by WinformsWizard** [comment]
Hi. I am using `OpenTK.NetStandard` and .NET Core 3.1 on Ubuntu 20.04. I followed the lesson "Creating a window" exactly, and my window is not opening. Here is my source code: https://git.io/JUyKZ I added a few `Console.Writeline` calls, and it appears that it is crashing when creating a new window. `OpenTK.NetStandard' worked fine on my old Linux laptop, so I don't know what is wrong. Any help would be appreciated 😃
